DOC-1225 RS reorg: Databases section (#1839)
RedisLabs/redislabs-docs: content/rs/databases/active-active/data-types/sets-active-active.md

---
Title: Sets in Active-Active databases
linkTitle: Sets
description: Information about using sets with an Active-Active database.
weight: $weight
alwaysopen: false
categories: ["RS"]
aliases: [
    /rs/developing/crdbs/developing-sets-crdb/,
    /rs/references/developing-for-active-active/developing-sets-active-active/,
    /rs/references/developing-for-active-active/developing-sets-active-active.md
    /rs/databases/active-active/data-types/sets-active-active.md,
    /rs/databases/active-active/data-types/sets-active-active/,
]
---

> Redis Sets are an unordered collection of Strings. It is possible to

Reviewer: ```suggestion
A Redis set is an unordered collection of strings. It is possible to
```
